frontend/www/js/omegaup/components/arena/RunDetails.vue

```diff
@@ -297,5 +330,34 @@ export default class ArenaRunDetails extends Vue {
     const visible = this.groupVisible[group];
     this.$set(this.groupVisible, group, !visible);
   }
+
+  showDataCase(
+    cases: types.ProblemCasesContents,
+    caseName: string,
+    caseType: string,
+  ): string {
+    return cases[caseName] ? cases[caseName][caseType] : '-';
+  }
+
+  showDiff(cases: types.ProblemCasesContents, caseName: string): string {
+    if (!cases[caseName]) {
+      return '-';
+    }
+    const result = diff(`${cases[caseName].out}`, `${cases[caseName].output}`);
```

Comment:
```suggestion
    const result = diff(cases[caseName].out, cases[caseName].output);
```

acá la interpolación es innecesaria porque no se está concatenando con nada.

Reply:
Cierto, se quita